index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+  <title>IF-JS</title>
+</head>
+<body>
+</body>
+</html>
```

Comment:
Looks like you don't know how to use prettier.

Reply:
sorry, I forget corrected.

Reply:
It is ready.